challenge-02/README.md

```diff
@@ -58,8 +72,33 @@ Crie uma função com as seguintes características:
 5. Se nenhum argumento for passado, retorne o valor booleano `false`.
 6. E ainda, se nenhuma das condições acima forem atendidas, retorne `null`.
 */
-?
+
+function	recebeargumentos( a , b , c ) {
+	if( a !== undefined && b === undefined && c === undefined ) {
+		return a;
+	} else if( b !== undefined && a === undefined && c === undefined ) {
+		return b;
+	} else if( c !== undefined && a === undefined && b === undefined ) {
+		return c;
+	} else if( a === undefined ) {
+		return b + c;
```

Comment:
-1
Aqui você está dizendo:

> Se o primeiro parâmetro não for passado, some os outros dois.

Entende que não faz sentido? Se o primeiro parâmetro não for passado, então os outros dois também não serão :)